service/application/pom.xml

```diff
@@ -4,11 +4,11 @@
     <parent>
         <groupId>com.solace.maas</groupId>
         <artifactId>maas-event-management-agent-parent</artifactId>
-        <version>1.6.4-SNAPSHOT</version>
+        <version>1.6.5-SNAPSHOT</version>
```

Comment:
I don't think we need any of these 1.6.4 to 1.6.5 changes. As soon as we run the release, it will update all these versions to the requested number, e.g. 1.6.5 or 1.7.0 or 2.0.0. Right @gregmeldrum ?

Reply:
Correct, it should automatically update